cmd/cli/plugin/package/test/lib/package_plugin.go

```diff
@@ -120,17 +120,14 @@ func (p *packagePlugin) AddRepository(o *tkgpackagedatamodel.RepositoryOptions)
 	if o.Namespace != "" {
 		cmd += fmt.Sprintf(" --namespace %s", o.Namespace)
 	}
-	if o.CreateNamespace {
-		cmd += fmt.Sprintf(" --create-namespace")
-	}
 	if o.Wait {
 		cmd += fmt.Sprintf(" --wait=true")
 	}
 	if o.PollInterval != 0 {
-		cmd += fmt.Sprintf(" --poll-interval %s", o.PollInterval)
+		cmd += fmt.Sprintf(" --wait-check-interval %s", o.PollInterval)
 	}
 	if o.PollTimeout != 0 {
-		cmd += fmt.Sprintf(" --poll-timeout %s", o.PollTimeout)
+		cmd += fmt.Sprintf(" --wait-timeout %s", o.PollTimeout)
```

Comment:
Question: Why the renaming?

Reply:
The wait flags have been renamed in `kctrl` to make it consistent with other Carvel tools.